v2/netconf/ops/sessionfactory.go

```diff
@@ -10,14 +10,14 @@ import (
 
 // Defines a factory method for instantiating netconf sessions.
 
-// NewRPCSession connects to the  target using the ssh configuration, and establishes
+// NewSession connects to the  target using the ssh configuration, and establishes
```

Comment:
Assume this break all existing users of the lib?

Reply:
No. Has no impact on netconf users.

Reply:
Sorry - it's only the comment that's been updated.